fetchcode/package.py

```diff
@@ -327,3 +328,26 @@ def get_rubygems_data_from_purl(purl):
         download_url=download_url,
         **purl.to_dict(),
     )
+
+
+@router.route("pkg:brew/.*")
+def get_homebrew_data_from_purl(purl):
+    """
+    Generate `Package` object from the `purl` string of brew type
+    """
+    purl = PackageURL.from_string(purl)
+    name = purl.name
+    base_path = "https://formulae.brew.sh/api/formula"
+    api_url = f"{base_path}/{name}.json"
+    response = get_response(api_url)
+    declared_license = response.get("license") or None
+    homepage_url = response.get("homepage")
+    version = response.get("versions") or None
+    download_url = response.get("urls")["stable"]["url"]
```

Comment:
What is response does not have "URLs" ?

Reply:
Should I add `try and catch` or would you suggest something else?

Reply:
The documentation of the homebrew json api tells that the `download URL` will  be present inside a `stable` of `URLs`, refer [here](https://formulae.brew.sh/docs/api/). In the docs, as they have mentioned, if we are using the base path as `https://formulae.brew.sh/api/formula.json`, then the `download URL` will be present inside the `stable` of `urls`, so I thought of leaving the download URL as such. Please do suggest what should I add it to cover the edge cases, so that it does not fail.

Reply:
Look how these things are done in the above block of code

Reply:
Ok, will look into it. Thanks for the feedback, will do the changes by tomorrow.